server/routes/adjudicationForReport/formsTab/formsTabView.ts

```diff
@@ -0,0 +1,54 @@
+import { Request, Response } from 'express'
+import moment from 'moment/moment'
+import ReportedAdjudicationsService from '../../../services/reportedAdjudicationsService'
+import UserService from '../../../services/userService'
+import adjudicationUrls from '../../../utils/urlGenerator'
+import { hasAnyRole, momentDateToDatePicker } from '../../../utils/utils'
+import { DISFormfilterFromUiFilter } from '../../../utils/adjudicationFilterHelper'
+import config from '../../../config'
+
+export default class FormsTabRoute {
+  constructor(
+    private readonly reportedAdjudicationsService: ReportedAdjudicationsService,
+    private readonly userService: UserService
+  ) {}
+
+  view = async (req: Request, res: Response): Promise<void> => {
+    const { user } = res.locals
+    const userRoles = await this.userService.getUserRoles(user.token)
+    if (config.formsTabFlag !== 'true' || !hasAnyRole(['ADJUDICATIONS_REVIEWER'], userRoles)) {
+      return res.render('pages/notFound.njk', { url: req.headers.referer || adjudicationUrls.homepage.root })
+    }
+
+    const { chargeNumber } = req.params
+    const { reportedAdjudication } = await this.reportedAdjudicationsService.getReportedAdjudicationDetails(
+      chargeNumber,
+      user
+    )
+
+    const prisoner = await this.reportedAdjudicationsService.getPrisonerDetails(
+      reportedAdjudication.prisonerNumber,
+      user
+    )
+
+    const filter = DISFormfilterFromUiFilter({
+      fromDate: momentDateToDatePicker(moment().subtract(6, 'months')),
+      toDate: momentDateToDatePicker(moment()),
+      locationId: null,
+    })
+    const results = (await this.reportedAdjudicationsService.getAdjudicationDISFormData(user, filter, false)).filter(
```

Comment:
Using same endpoint as the DIS 1/2 pages use which return the latest `dateTimeOfIssue`, see [Jira comment](https://dsdmoj.atlassian.net/browse/NN-5381?focusedCommentId=321714) that this may need to change